### PR TITLE
KNifantiev

### PR DESCRIPTION
**Pull request by Serpenter**

### CODE REVIEW

File: Kostiantyn_Nifantiev/02/Matrix/Matrix.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from MatrixProcessingFunc import *
```
[the remark this answers is not in the document]
Comment (Serpenter) [reply Kostiantyn_Nifantiev/02/Matrix/Matrix.py]: Fixed

File: Kostiantyn_Nifantiev/02/Matrix/MatrixProcessingFunc.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+def get_matrix():
+    """
+    Getting matrix from user. Returns it in form capable for further proceeding 
+    """
+    input_matrix = input('Enter yor matrix with "," as separator between numbers and ";" between rows:\n')
+    input_matrix = "\n".join(input_matrix.split(';'))
+    return input_matrix
+
+
+def read_matrix_rows(matrix):
```
[the remark this answers is not in the document]
Comment (oleksandr-medviediev) [reply Kostiantyn_Nifantiev/02/Matrix/MatrixProcessingFunc.py]: It should parse matrix and then recombine everything. I've explained this.

**Comment by Serpenter** [comment]
> Create separate folders for the 2nd and the 3rd batches.
They are separate. 3rd batch in folder 02, 2nd in 01, first in 00. I have a confusing in this case file/folder naming habit formed as result of pain gained from naive 1-2-3 file/folder counting in some use cases. Should i rename all of them?

> Bad using of module structures. 
Anything else besides dangerous "from x import *" construction?

> Read tasks carefully! Try to avoid doing excess stuff while you don't need it!
Is it about test-case scripts? Should i skip them in commits?

File: Kostiantyn_Nifantiev/02/Matrix/MatrixProcessingFunc.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (Serpenter) [reply Kostiantyn_Nifantiev/02/Matrix/MatrixProcessingFunc.py]: Fixed

**Pull request merged**